src/components/Trade/TradeInput.tsx

```diff
@@ -556,6 +601,39 @@ export const TradeInput = () => {
     [isSwapperApiPending, isSellAction],
   )
 
+  const ManualReceiveAddressEntry: JSX.Element = useMemo(() => {
+    return (
+      <FormControl>
+        <FormLabel color='gray.500' w='full'>
+          {translate('trade.receiveAddress')}
+        </FormLabel>
+        <AddressInput
+          showQr={false}
```

Comment:
Do we plan to eventually bring QR support here or is this deemed as too risky WRT validation?

Reply:
side note it seems wierd to pass in explicit false to disable something via props, i'd expect the converse `disableQr`

Reply:
@gomesalexandre I wanted to take it one step at a time, and QR wasn't a part of the spec, though there is no reason we couldn't or shouldn't do it.

We would have to fix the current handler though, as this `AddressInput` isn't in a modal (where it was originally used), so scanning won't work without some additional generalising of the component.

@woodenfurniture you're right, that is weird. Fixed up in https://github.com/shapeshift/web/pull/4479/commits/ae7253c23f0352f585bab637223ea053ef5e3843 by renaming it to `enableQr`, and having is `false` unless explicitly specified.

Reply:
Sounds good to me, we might want to remove the prop when all `<AddressInput />` should show QR code scanning icon, but that makes total sense for now!